linux_sql/sql/ddl.sql

```diff
@@ -18,23 +21,4 @@ CREATE TABLE IF NOT EXISTS PUBLIC.host_usage (
   disk_io INT,
   disk_available INT,
   FOREIGN KEY(host_id) REFERENCES host_info(id)
-);
-INSERT INTO host_info (
-  hostname, cpu_number, cpu_architecture,
-  cpu_mhz, L2_cache, total_mem, "timestamp"
-)
-values
-  (
-    'spry-framework-236416.internal',
-    1, 'x86_64', 2300.000, 256, 601324,
-    '2019-05-29 16:53:28'
-  );
-INSERT INTO host_usage (
-  "timestamp", host_id, memory_free,
-  cpu_idle, cpu_kernel, disk_io, disk_available
-)
-VALUES
-  (
-    '2019-05-29 16:53:28', 1, 256, 95,
-    0, 0, 31220
-  );
\ No newline at end of file
+);
```

Comment:
Does the host_usage table have an id? Where is it? 
You can set the primary key constraint to be:
```
PRIMARY KEY (timestamp, host_id)
);
```

Reply:
Will add a primary key for host_usage. I